internal/worker/sshserver/server.go

```diff
@@ -143,6 +136,28 @@ func NewServerWorker(config ServerWorkerConfig) (worker.Worker, error) {
 	return s, nil
 }
 
+func (s *ServerWorker) NewJumpServer() *ssh.Server {
```

Comment:
missing godoc

Reply:
I'm gonna try remember to fix this in a follow-up.